File: .sdk/tm/ts/test/exists.test.ts

import { test, describe } from 'node:test'
import { equal } from 'node:assert'


import { ImgflipSDK } from '..'


describe('exists', async () => {

  test('test-mode', async () => {
    const testsdk = await ImgflipSDK.test()
    equal(null !== testsdk, true)
  })

})

File: lua/test/exists_test.lua
-- Imgflip SDK exists test

local sdk = require("imgflip_sdk")

describe("ImgflipSDK", function()
  it("should create test SDK", function()
    local testsdk = sdk.test(nil, nil)
    assert.is_not_nil(testsdk)
  end)
end)
